### PR TITLE
Issue #448: resolve more codenarcs

### PR DESCRIPTION
**Pull request by rnveach**
Issue #448

### CODE REVIEW

File: checkstyle-tester/diff.groovy [hunk 1]
```diff
@@ -861,3 +861,7 @@ class CheckstyleReportInfo {
         this.commitTime = commitTime
     }
 }
+
+class Constants {
+    static final TWO = 2
```
Comment (romani) [inline checkstyle-tester/diff.groovy:866]: Do we really need to make our code terrible?
I would rather keep suppression or disable rule

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Comment by rnveach** [comment]
https://app.travis-ci.com/github/checkstyle/contribution/jobs/588259391
Retrying.

**Comment by rnveach** [comment]
Failure is stable but no reason why is given. Only change is the class loading now.

**Review by romani — APPROVED** [review]

**Comment by rnveach** [comment]
It is passing now.

**Pull request merged**